tests/integration/test_crunch_cube.py

```diff
@@ -1407,275 +1405,6 @@ def test_subtotals_proportions_two_transforms_missing_included(self):
                                   include_transforms_for_dims=[0])
         np.testing.assert_almost_equal(actual, expected)
 
-    def test_labels_on_2d_cube_with_hs_on_1st_dim(self):
```

Comment:
what's up with all this delete? did it move somewhere else? wrong branch or something?

Reply:
It moved to a separate file (test headings and subtotals). It was just a preparation part for this PR, you can see it if you go commit by commit.